### DISCUSSION TITLE
Juvelnile Bull, убит вместо убили

### DISCUSSION THREAD

**Discussion by registant** [mindryder/ATLAS-RU#4] [НАЙДЕННЫЙ ПРОПУЩЕННЫЙ ПЕРЕВОД]
![Screenshot_1325](https://user-images.githubusercontent.com/123338303/214049163-573642c9-6096-4305-8384-ae4fdc33ae48.jpg)

**Comment by mindryder** [comment]
Дело в том что в игре одно значение killed - для убийства кого-то, и для такого случая. Надо ещё подумать как перевести чтобы склонялось в двух вариантах, либо в скобочках сделать. Либо вообще смайликом черепа сделать (правда он будет отображаться только если прикручен главный шрифт). Вообщем надо ещё подумать решить это место перевода.

![AtlasGame_R8wICF6tZt](https://user-images.githubusercontent.com/66848160/214050694-912e3156-0b3a-408c-b0ab-51cd2b7b003d.png)

**Comment by mindryder** [comment]
Пришла идейка, сейчас попробую.

**Comment by mindryder** [comment]
![AtlasGame_lSRfYvjAWZ](https://user-images.githubusercontent.com/66848160/214053995-e07640ac-fa1c-400e-88d1-49209437b46f.png)
![ACDSeeQVUltimate15_zDqSvWPQq5](https://user-images.githubusercontent.com/66848160/214054118-329e3915-1140-4bc8-96f7-fbb4c5d7e3e8.png)

Пойдет.

**Comment by registant** [comment]
Огонь

**Comment by registant** [comment]
Обновил до 1,3, но логи и смерти частично все еще на английском(
![Screenshot_1341](https://user-images.githubusercontent.com/123338303/214251229-40bfa403-e977-42d4-9b0f-c8320e0bc210.jpg)
![Screenshot_1342](https://user-images.githubusercontent.com/123338303/214252318-b775a941-978f-4c21-b0fa-6f62ee742155.jpg)
![Screenshot_1343](https://user-images.githubusercontent.com/123338303/214252470-abd737d0-c0e5-43bd-8921-612e23d7de9a.jpg)
![Screenshot_1344](https://user-images.githubusercontent.com/123338303/214256884-0b03aed7-49ad-42a6-a8a1-3402a36b0a01.jpg)
